### DISCUSSION TITLE
SubtitleEditBeta 587 Bug

### DISCUSSION THREAD

**Discussion by forapreder** [SubtitleEdit/subtitleedit#7237] [General]
SubtitleEditBeta from 570 to 587 Bug: After recognizing subtitles to text from a video file, pressing the OK button does not save the text.

**Comment by niksedk** [comment]
Where and what are you doing?
Could you provide a screenshot or two...

**Comment by forapreder** [comment]
File->Import->Subtitle from video file->Transport steam subtitle chooser->Start OCR->OK
The text does not appear in the subtitle window SubtitleEditBeta from 570 to 587 and 588 too.
SubtitleEditBeta 540 was OK.
![1001](https://github.com/SubtitleEdit/subtitleedit/assets/60479869/35884b19-d947-4aba-8ccb-bfc11a209e8d)
![1002](https://github.com/SubtitleEdit/subtitleedit/assets/60479869/8fae2597-d329-45c3-a090-09172fe5e603)
![1003](https://github.com/SubtitleEdit/subtitleedit/assets/60479869/e95cfe6c-31c9-4bea-a69b-d7719984c8b2)
![1004](https://github.com/SubtitleEdit/subtitleedit/assets/60479869/d7082750-57fc-4621-b3c3-376266459b3d)

**Comment by niksedk** [comment]
Hopefully better now: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.13/SubtitleEditBeta.zip
?

**Comment by forapreder** [comment]
The bug has been fixed.
Thank you very much!
![1001](https://github.com/SubtitleEdit/subtitleedit/assets/60479869/40fa1a19-f02c-4f05-b310-9995afcd542e)
